### ISSUE TITLE
ConjugacyClassesSubgroups(PO(5,3) x PO(5,3)) gives unexpected error in 4.11.1 (maybe tomlib related)

### ISSUE THREAD

**Issue by jackschmidt** [gap-system/gap#4903]
`ConjugacyClassesSubgroups(DirectProduct(POmega(5,3),POmega(5,3)));` gives an unexpected error in GAP 4.11.1 (with tomlib loaded).

In 4.11.1 without tomlib loaded, the calculation finishes successfully after about an hour.

In 4.dev with tomlib loaded, the calculation finisheses successfully after about an hour.

When the error shows up in 4.11.1, it shows up very quickly.

### Observed behaviour

```
 ┌───────┐   GAP 4.11.1 of 2021-03-02                                                                                  
 │  GAP  │   https://www.gap-system.org                                                                                
 └───────┘   Architecture: x86_64-pc-linux-gnu-default64-kv7                                                           
 Configuration:  gmp 6.2.1, GASMAN, readline                                                                           
 Loading the library and packages ...                                                                                  
 Packages:   AClib 1.3.2, Alnuth 3.1.2, AtlasRep 2.1.0, AutoDoc 2020.08.11, AutPGrp 1.10.2, Browse 1.8.11,             
             CRISP 1.4.5, Cryst 4.1.23, CrystCat 1.1.9, CTblLib 1.3.1, FactInt 1.6.3, FGA 1.4.0, Forms 1.2.5, 
             GAPDoc 1.6.4, genss 1.6.6, IO 4.7.0, IRREDSOL 1.4.1, LAGUNA 3.9.3, orb 4.8.3, Polenta 1.3.9, 
             Polycyclic 2.16, PrimGrp 3.4.1, RadiRoot 2.8, recog 1.3.2, ResClasses 4.7.2, SmallGrp 1.4.2, 
             Sophus 1.24, SpinSym 1.5.2, TomLib 1.2.9, TransGrp 3.0, utils 0.69
 Try '??help' for help. See also '?copyright', '?cite' and '?authors'
gap> ConjugacyClassesSubgroups(DirectProduct(POmega(5,3),POmega(5,3)));
Error, <gens> must be elements in <M> at /usr/lib/gap/lib/magma.gi:543 called from
Subgroup( myfgi[5], inducedfactorautos( k.normalizer, myfgi[4], myfgi[3] ) ) at /usr/lib/gap/lib/grplatt.gi:2755 called from
SubdirectSubgroups( n ) at /usr/lib/gap/lib/grplatt.gi:2926 called from
SubgroupsTrivialFitting( G ) at /usr/lib/gap/lib/grplatt.gi:914 called from
LatticeSubgroups( G ) at /usr/lib/gap/lib/grplatt.gi:208 called from
<function "ConjugacyClassesSubgroups group">( <arguments> ) 
 called from read-eval loop at *stdin*:1
you can 'quit;' to quit to outer loop, or
you can 'return;' to continue 
brk> 
gap> 
```

If I try `gap -A` but also `LoadPackage("tomlib");` then the same error occurs at the same speed.

I tried `gap -A` without tomlib, and it works (after about an hour).

### Expected behaviour

After a an hour, compute the subgroups. GAP handles similarly sized groups of the form G x G.

### Copy and paste GAP banner (to tell us about your setup)

Banner is above. This is the arch release. Here is a reproducible version for ubuntu:

```
$ docker run -it --rm ubuntu:latest
# apt-get update && apt-get install -y --no-install-recommends gap-table-of-marks gap-character-tables gap gap-smallgrp
# gap
 *********   GAP 4.11.1 of 2021-03-02
 *  GAP  *   https://www.gap-system.org
 *********   Architecture: x86_64-pc-linux-gnu-default64-kv7
 Configuration:  gmp 6.2.1, GASMAN, readline
 Loading the library and packages ...
 Packages:   AtlasRep 2.1.0, CTblLib 1.3.1, GAPDoc 1.6.4, SmallGrp 1.4.1, TomLib 1.2.9
 Try '??help' for help. See also '?copyright', '?cite' and '?authors'
gap> ConjugacyClassesSubgroups(DirectProduct(POmega(5,3),POmega(5,3)));
Error, <gens> must be elements in <M> at /usr/share/gap/lib/magma.gi:543 called from
Subgroup( myfgi[5], inducedfactorautos( k.normalizer, myfgi[4], myfgi[3] ) 
 ) at /usr/share/gap/lib/grplatt.gi:2755 called from
SubdirectSubgroups( n ) at /usr/share/gap/lib/grplatt.gi:2926 called from
SubgroupsTrivialFitting( G ) at /usr/share/gap/lib/grplatt.gi:914 called from
LatticeSubgroups( G ) at /usr/share/gap/lib/grplatt.gi:208 called from
<function "ConjugacyClassesSubgroups group">( <arguments> )
 called from read-eval loop at *stdin*:1
you can 'quit;' to quit to outer loop, or
you can 'return;' to continue
brk>
gap>
```

**Comment by fingolfin** [comment]
This error does not seem to be present in the GAP master branch. 

We really need a new release *sigh*

**Comment by fingolfin** [comment]
@jackschmidt thanks for the report! I am gonna close this for now as "is fixed in master / will be fixed in GAP 4.12", which is due to be released in mid August
